geocase-linkage/requirements.txt

```diff
@@ -1,2 +1,3 @@
 kafka-python==2.0.2
 requests==2.31.0
+shapely==2.0.4
```

Comment:
Do you need shapely?

Reply:
Yeah it's used to to parse the georeference info (line 93)